app/views/projects.html

```diff
@@ -25,9 +25,13 @@ <h2 class="text-center">Loading...</h2>
                         <h1>My Projects</h1>
                         <div class="projects-options">
                           <div class="projects-add" ng-if="canCreate">
-                            <a href="create-project" class="btn btn-md btn-primary">
+                            <button ng-click="createProject()" class="btn btn-md btn-primary">
+                              <span class="fa fa-plus"></span>
                               Create Project
-                            </a>
+                            </button>
+                            <origin-modal-popup shown="showNewProjectPanel" modal-title="Create Project" on-close="closeNewProjectPanel()">
```

Comment:
Right, understood. I suggest we should use `<` or `=` like the other components. I'd at least like to be consistent with callbacks inside the same component. Here `shown` is `<` and `onClose` is `&` in the same component.

Sorry, I missed this reviewing https://github.com/openshift/origin-web-common/pull/135

(As an aside, I find callbacks in Angular components generally weird and confusing...)